pysat/tests/test_meta.py

```diff
@@ -1076,6 +1173,8 @@ def test_get_variable_name_case_preservation(self):
         assert ('NEW2' == self.meta.var_case_name('NEW2'))
 
     def test_get_attribute_name_case_preservation(self):
+        """Test that getting attribute names preserves the case.
+        """
```

Comment:
This test needs a check that the attribute name is stored as 'YoYoYO', based on the name. Currently tests preservation of case for input meta values, as well as tests case independence for accessing labels.

Reply:
Line 1198 has the goods.

Reply:
Perhaps, 'Test that meta labels and values preserve the case'